policy/modules/contrib/wireguard.if

```diff
@@ -0,0 +1,41 @@
+## <summary>policy for wireguard</summary>
+
+########################################
+## <summary>
+##      Execute wireguard_exec_t in the wireguard domain.
```

Comment:
Please use tabs for formatting.

Reply:
There is an inconsistency in using the base identifier. Can just `wireguard` be used in all types or is there something specific about wg-quick?

Reply:
Fixed, thank you